.github/workflows/release.yml

```diff
@@ -393,29 +393,37 @@ jobs:
           docker run --network ${{ job.container.network }} --rm test-${{ matrix.distro }}
 
   smoke-tests:
-    if: startsWith(github.ref, 'refs/tags')
     needs:
       - goreleaser
     runs-on: ubuntu-latest
+    permissions:
+      id-token: write   # AWS OIDC JWT
+      contents: read    # actions/checkout
 
     steps:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 1
 
-      - name: Run tests
+      - uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: arn:aws:iam::754489498669:role/ecr_rw_tyk
+          role-session-name: cipush
+          aws-region: eu-central-1
+          # Don't mask to pass it across job boundaries
+          mask-aws-account-id: false
+
+      - uses: aws-actions/amazon-ecr-login@v1
+        id: ecr
+
+      - name: "Run ci/tests"
```

Comment:
These tests are outside the templates and perhaps we could remove this altogether.